app/models/sqlalchemy/jobstore.py

```diff
@@ -0,0 +1,108 @@
+"""
+jobstore.py
+Tools for storing jobs from MolSSI SEAMM
+
+Has the SEAMM datastore class.
+"""
+
+import os
+import glob
+
+from contextlib import contextmanager
+from datetime import datetime
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker, relationship
+
+from .models import Base, Flowchart, Job, Project, JobProject
+from .util import process_flowchart, process_job
+
+
+class DataStore():
+    def __init__(self, db_path, db_type='sqlite'):
+        self.db_location = os.path.abspath(db_path)
+
+        # TODO Check if exists. If exists, check structure to see if it is correct.
+        # If exists and not correct format - error and exit.
+        if os.path.exists(self.db_location):
+            os.remove(self.db_location)
+
+        engine = create_engine(db_type + ':///' + self.db_location)
+        Base.metadata.create_all(engine)
+
+        self.Session = sessionmaker(bind=engine)
```

Comment:
you have created the engine before in the `create_app` factory method, does this need to be removed?